Automattic/delphin
# i18n: Pull the translations from the delphin directory

This pull request updates Delphin to use delphin translations rather than Calypso ones.
#### Testing instructions
1. Run `git checkout update/translations`
2. Run `npm run clean:i18n-cache`
3. Start your server
4. Open the http://delphin.localhost:1337/pt-br
5. Confirm that the "About" page link says "Sobre"
#### Reviews
